packages/headless/src/app/analytics-middleware.test.ts

```diff
@@ -2,9 +2,9 @@ import {logSearchboxSubmit} from '../features/query/query-analytics-actions';
 import {
   executeSearch,
   ExecuteSearchThunkReturn,
-} from '../features/search/search-actions';
+} from '../features/search/legacy/search-actions';
 import {buildMockSearchAppEngine} from '../test/mock-engine';
-import {buildMockSearch} from '../test/mock-search';
+import {buildMockLegacySearch} from '../test/mock-search';
```

Comment:
This is the only test file that is likely to stay and die on the buildMock, as we're not porting the analyticsMiddleware to v3